### ISSUE TITLE
public_path error in ImageMeta.php 

### ISSUE THREAD

**Issue by romatexdev** [ralphjsmit/laravel-seo#87]
Good day! Can you tell me what the problem might be? I prepare the seo data and send it to the template. 
```
		$title = mb_strimwidth($this->name, 0, 70, "");
        $description = mb_strimwidth($this->description, 0, 500, ".");
        $imageUrl = $this->hasMedia('products') ? $this->getFirstMediaUrl('products', 'products-image') : null;
        return new SEOData(
            title: $title,
            description: $description,
            image: $imageUrl,
        );
```
And, as expected, I successfully receive them on the page. When checking the $imageUrl variable before sending it to the template, I also get the correct path. 
`        <meta name="image" content="https://test.ts/media/images/products/9244/5911/conversion/products-image-prothermix-skat-9-ke.webp">`

But laravel logs an error: We get this error when checking in the file ImageMeta.php this package. 
`ERROR: Path /home/testy/public_html/public/https://test.ts/media/images/products/9244/5911/conversion/products-image-prothermix-skat-9-ke.webp is not a file. {"exception":"[object] (Exception(code: 0): Path /home/testy/public_html/public/https://test.ts/media/images/products/9244/5911/conversion/products-image-prothermix-skat-9-ke.webp is not a file. at /home/testy/public_html/vendor/ralphjsmit/laravel-seo/src/Support/ImageMeta.php:18)`
```
//vendor/ralphjsmit/laravel-seo/src/Support/ImageMeta.php
public function __construct(string $path)
    {
        $publicPath = public_path($path);

        if (! is_file($publicPath)) {
            report(new Exception("Path {$publicPath} is not a file."));

            return;
        }

        [$width, $height] = getimagesize($publicPath);

        $this->width = $width;
        $this->height = $height;
    }
```

Tell me why is the check not going through in the file ImageMeta.php?

**Comment by ralphjsmit** [comment]
Hello Ramis, thank you for your message! I think the reason was that the ImageMeta works fine if you pass a path, and then it tries to find the image size from that. If you pass a url, it would return and report it as invalid. In your, the report was not was not really necessary if `$path` is a valid URL, so I have just made a fix in 22bb096 that will return early if the path is a URL. That has been released in 1.6.4. Let me know if that fixes it!
